module/Application/src/Application/View/Helper/Acl.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Application\View\Helper;
+
+use Zend\View\Helper\AbstractHelper;
+
+
+class Acl extends AbstractHelper
+{
+
+    /**
+     * Acl
+     *
+     * @var \Zend\Permissions\Acl\Acl
+     */
+    protected $acl;
+
+    /**
+     * Get the current user's role.
+     *
+     * @return User|string
+     */
+    protected $role;
+
+    /**
+     * Check if a operation is allowed for the current role.
+     *
+     * @param string $operation Operation to be checked.
+     * @param string|ResourceInterface $resource Resource to be checked
+     *
+     * @return boolean
+     */
+    public function __invoke($operation, $resource)
```

Comment:
Since it's not possible to leave the resource empty here in any way, it is better to have the order of the parameters the other way around. So the order is the same as the `isAllowed` and other operations like `allow` and `deny` in the zf2 ACL implementation.